version.go

```diff
@@ -1,6 +1,8 @@
 package ydb
 
-import "github.com/ydb-platform/ydb-go-sdk/v3/internal/meta"
+import (
+	"github.com/ydb-platform/ydb-go-sdk/v3/meta"
+)
 
 // Version reports current version of sdk
 const Version = meta.Version
```

Comment:
Now version available in two places:
1. There
2. meta/version.go

I think it should be removed from public available in meta/version.go

Reply:
You are right. But splitting version helps to exclude imports each other. Root package cant be imported from sub-packages because it provokes mutual imports and code cant be compiled.
Root version stayed for backward compatibility
meta/version.go - for newest usage. May be I didnt right

Reply:
What about - hide generated version to internal (as in main)?

I doesn't like many deprecated code with renaming, backward compatible and so on.
It make more difficult to understand public interface.

Rename/move possible, if it has strong reason only.

Reply:
Ack